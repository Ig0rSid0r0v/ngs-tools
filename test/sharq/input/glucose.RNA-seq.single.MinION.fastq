@a69dd3c2-c98f-4f17-9da5-fe64f97494f6_Basecall_1D_template:1D_000:template
AGAUUUAAUCCAAGAGAAGCAAAAAAAGCUCUAUAAAAUAUCAUGCAAUUACAUUACCAUCGCUUCAACUGCGCUCUAAUAACGUCAGCUUUGGCUAAUUACUUCCUUUUCCCAAUAUACCCUUUUCCUGAGGUACCAUUGGUUUCAUGCCAAUCCUGUCAGAGUGAACAGAUAAAUAUAGCCAAUUGUGAUACGUUAGUCUGGAACUUAACUAUUGGUAGUUUGAAGACUGGUGCUACGUUAAUGUUAAAAGGAAUCAACGGGUCACUAACUAUUUAACGCUACAAAUUUAACCUCAUUCGCUGCUGAUCCUGGAGUCCAACACAGAUUCUUUUGAACCCUACAGAGUUUGACAAUCUUGACUUCUUGCUUCAUUUUGGGUUUUACAGAGCUGAAUUAUAAGCUAUAAACUGAUUACUUCCCCGCCAUCUCCCUCCAGUUUACUUCACAAUAUAAAUACACUGCUAACAUUUAUUCCGACACCACUUUAUACAAUCUGUCGAUGGAUUCAGCCUAAAAGUUAACGUGUUUCAACGUCAAUAUAUACUGAAAUUAACCCAUUGAUCAAAUCUCCAGUUGAACAGCUCAGCGAUUUUUACAAUUCGUUUUACAACGGUAACCAGACUAAAAUCACCUUGGUGACUUGGUUUGGGCAAACAUAUCAGUUUGACCGAUGUCCACUCUGUUUCCUUCGUUAACUUGCAAAAGAUUAACUUCAUUAGGUCACUUUUAACAUUUCCAUCUCAAGUUUGAAUUUCUAAGCUAACACCGCCAAACCUUCAGUAUCGUUUCAAUGACUACUUGAAGAACUUUGUCGUUCUCUUCGAAUUGUCAACCACCAUAGGUGGUGUGCUUGUCGUCGCUAAUAUUAGGUCUUUACAAAAAAUUGGUGGUUCGACCAACCUAACAACCAUUGGCAGUACUUUGGUAGUUGUUAUUGGUAACUUUUUCUCCUGAACCUAGACUUUCUUUGAAGUUGGUGGCGCAGAUGUCGAAUAAGUCAAGCAAUUUCCUGUAAUGCUUUGAAAGCUUUUGCAAAAAGAAAGCUAUCAAGGGUGAAUCUUUUGUUUGCAAAAUGGUGCUACCCACAUACUUGUUAAACUCGCCUUUCCAAUACAAUCAAGCCAAACUACUGCCAAGGUUUUCCAAGUCAUUUAAGGCCGGGAAAGAAGGUUUUGCGAUAUCAAGGCUGCUGCUUUCUGCCCCUCUAGUGUUUUAGUUUCUAGCAAUUCCAGCUCUAGCUUAAGAGAGUUCCGACAAUGCCGCUAUCAUGGCACCAAUUGGCCAAACAACCCUUUGGUCGGUUUUUUGACGGCAAUCAUCAUGUUCUAUAAUGUAAUAGAAUGAAGAAAAAUUCUCAUUUUGACUAGUACCUGUCAUCACGACACGAACAAAUAAACAUUUAUUUAAAAAUUUUAUGUAUUCAAAUAUUUUUCGUGGGAAAGGAUAAAGUAAACCAA
+
2$&@/A6D:48AN)(('2C:HcbT_T7*&("&"%,haC?C?9;>?<PO9?3.)'+")/*=95)59')0&@(#+*:"C*1:-C7=;=@S69?QE_OI(-#1"$0E3C,=45*Q:9+04=24QR*4>?,4+'"&.06:7./-1)(,'+)/,62&*";0/""$$""'#%$+)-6207YJEB<B?AW"(64$586A0'""*-->;4MEO4,2-20%79HA7+$&*+RZBYUNI/*D'31(-$#+6PDGV-*<5@;4$,-\)"#4&&8I+&%*/WI033@@6@F9945(#*?I6)7@1@@129-G#*+:96JLU..)-*/010IJ?4814;=C@G);31503'$D7$(&'*.0;ETadb,[$"/Z7VCG1#066*@<1429@=?XG--*4%-/04""7)(1%(,$'+%$-8/HS@0=B@H=3%,&#6WaNc<?.("FW005>3.A96/.-")).IG5777N5K8;&#*(-+.3'0)&%1$.(("-""/50262K<*"%&''4"/""*"(/=I.@+#0_B9,8@93.D+*"*57/L6(*?,942>=+/*/00"6;8ZQ,3:".V7"0,%&.$3cR:/913(7K?FMD7,/"5--,#+*'C152&(>D5/*&-A2+"%"5*=JBIC.((,13@J\^B7(#+0*/%+*0015>CI?5>5O/<7?@"++-)4-:6,"6$""(')+,2@.,((:<?[+00+,+)-D;'"U962/OTE2;AHA8*&4)*#)"'="""%-;FADE>F2(1*C>:-4//@19QWX^B<:4"'2F*,"-3"%))&(.BkSM2<:90HP78,6P,")",00B5532<?>;D9@H7-+0&3),-8%&$('"$%%*N14/+2)1&+1'"#5;HS?3.%($6=;-#./'*'02-*&'*"#6#"'-C06:V][QUBKW^WQZ+%2**'"#*F=>\LH_QUD1.+.1-)67"$0+68ICENK8=84)15,<471*@K=:4"'9-9:eYER-BG?,811"/5<2F<6##(3)B2*$"2<>MKaVceJE7G/5%+//%CE202.//9CZLJ7%-+4+DF50*484J(AIdkW:L>A,*%.+02/4EX95?O9E$28GN"/*H13Zic72L?GE6;.-;,#0/""+0-51CGS2'%""-9*.4-:.;%))90>1.I9;S9/6#*'(*"$7(,5>:NG38<684(3/1:6&&6J.>,%2;O4334+27@0("%5/RA5#(7*AAM76;1*9/.*81+2N?,&52:8D-@;E5G22<*903*/9-0(''112*3/5./=++'*7426+/0(*+0A&"&1*)94611GNR?L:-@G=>7M=K<dJ64N)H2(1.)BA/&G96=IRkkA8*L9aY?GNB""""38+;>-/=015B3&"38HD?MJ=Bjia>D='0,65CE,4'"'/0TOROJT?9@2'*01*,//:HL?:LV-7S4&%+)+,>cOAgfe@I?5/2*;;A8)-9R?66UWJ>-')")/&A61/53-/;0""""""#""
