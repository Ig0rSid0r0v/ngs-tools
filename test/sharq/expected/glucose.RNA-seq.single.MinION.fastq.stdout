[
{
"row_id": 1,
"BASE_COUNT":1460,
"BIO_BASE_COUNT":1460,
"CHANNEL":0,
"CMP_BASE_COUNT":1460,
"COLOR_MATRIX":[0, 1, 2, 3, 4, 1, 0, 3, 2, 4, 2, 3, 0, 1, 4, 3, 2, 1, 0, 4, 4, 4, 4, 4, 4],
"CSREAD":"32230030320102222023100000023222333000333213131030311303101323320210121333222303301312123200103230303120202000200103333100200020212201310130101002131301032021121222112011223300333323010301112331310321221020120301233010132100120221210113231310303110300020203210130012112301233003013323110030030102213023321321232021022120101111223022000120100231122210012110322012120220132021300010010003112223212030333023233300121230312020003301322200220121003120211103333003311121323011300330203211101120033311032211232310230212302300021030131110021013121033333312120030301001301232100322201210120112322123323000031103023100031101301301012212300032110201011212010100100310011333212100121032311201122211002020231030120131000223030120213032012112000301130020132221021001203002230232301110330100102021213323100210312123120120220120011231022220232030112101011013320110111132011231233230333032012200311000003010110102321010102301101013010312131200101321011033010130120000222021201023221200220012021010110333122311232033021210231030020211303132001200232000131000022002323321020011203220001100131000310113231001113312011030012233020020103311032102301001231213010201000201021213003020303002002202010001332333210203213213200221300022232111000321002232310302012322232320302222210203211031303323321310311010301030100110100200101230100000121303103213213110223330311303322031202200003022213000121232131021121321132111320110033001130033003000030003311330210033300002311002002023300213001010",
"CS_KEY":"T",
"CS_NATIVE":false,
"FIXED_SPOT_LEN":1460,
"MAX_SPOT_ID":1,
"MIN_SPOT_ID":1,
"NAME":"a69dd3c2-c98f-4f17-9da5-fe64f97494f6_Basecall_1D_template:1D_000:template",
"PLATFORM":"SRA_PLATFORM_OXFORD_NANOPORE",
"POSITION":[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127, 128, 129, 130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146, 147, 148, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160, 161, 162, 163, 164, 165, 166, 167, 168, 169, 170, 171, 172, 173, 174, 175, 176, 177, 178, 179, 180, 181, 182, 183, 184, 185, 186, 187, 188, 189, 190, 191, 192, 193, 194, 195, 196, 197, 198, 199, 200, 201, 202, 203, 204, 205, 206, 207, 208, 209, 210, 211, 212, 213, 214, 215, 216, 217, 218, 219, 220, 221, 222, 223, 224, 225, 226, 227, 228, 229, 230, 231, 232, 233, 234, 235, 236, 237, 238, 239, 240, 241, 242, 243, 244, 245, 246, 247, 248, 249, 250, 251, 252, 253, 254, 255, 256, 257, 258, 259, 260, 261, 262, 263, 264, 265, 266, 267, 268, 269, 270, 271, 272, 273, 274, 275, 276, 277, 278, 279, 280, 281, 282, 283, 284, 285, 286, 287, 288, 289, 290, 291, 292, 293, 294, 295, 296, 297, 298, 299, 300, 301, 302, 303, 304, 305, 306, 307, 308, 309, 310, 311, 312, 313, 314, 315, 316, 317, 318, 319, 320, 321, 322, 323, 324, 325, 326, 327, 328, 329, 330, 331, 332, 333, 334, 335, 336, 337, 338, 339, 340, 341, 342, 343, 344, 345, 346, 347, 348, 349, 350, 351, 352, 353, 354, 355, 356, 357, 358, 359, 360, 361, 362, 363, 364, 365, 366, 367, 368, 369, 370, 371, 372, 373, 374, 375, 376, 377, 378, 379, 380, 381, 382, 383, 384, 385, 386, 387, 388, 389, 390, 391, 392, 393, 394, 395, 396, 397, 398, 399, 400, 401, 402, 403, 404, 405, 406, 407, 408, 409, 410, 411, 412, 413, 414, 415, 416, 417, 418, 419, 420, 421, 422, 423, 424, 425, 426, 427, 428, 429, 430, 431, 432, 433, 434, 435, 436, 437, 438, 439, 440, 441, 442, 443, 444, 445, 446, 447, 448, 449, 450, 451, 452, 453, 454, 455, 456, 457, 458, 459, 460, 461, 462, 463, 464, 465, 466, 467, 468, 469, 470, 471, 472, 473, 474, 475, 476, 477, 478, 479, 480, 481, 482, 483, 484, 485, 486, 487, 488, 489, 490, 491, 492, 493, 494, 495, 496, 497, 498, 499, 500, 501, 502, 503, 504, 505, 506, 507, 508, 509, 510, 511, 512, 513, 514, 515, 516, 517, 518, 519, 520, 521, 522, 523, 524, 525, 526, 527, 528, 529, 530, 531, 532, 533, 534, 535, 536, 537, 538, 539, 540, 541, 542, 543, 544, 545, 546, 547, 548, 549, 550, 551, 552, 553, 554, 555, 556, 557, 558, 559, 560, 561, 562, 563, 564, 565, 566, 567, 568, 569, 570, 571, 572, 573, 574, 575, 576, 577, 578, 579, 580, 581, 582, 583, 584, 585, 586, 587, 588, 589, 590, 591, 592, 593, 594, 595, 596, 597, 598, 599, 600, 601, 602, 603, 604, 605, 606, 607, 608, 609, 610, 611, 612, 613, 614, 615, 616, 617, 618, 619, 620, 621, 622, 623, 624, 625, 626, 627, 628, 629, 630, 631, 632, 633, 634, 635, 636, 637, 638, 639, 640, 641, 642, 643, 644, 645, 646, 647, 648, 649, 650, 651, 652, 653, 654, 655, 656, 657, 658, 659, 660, 661, 662, 663, 664, 665, 666, 667, 668, 669, 670, 671, 672, 673, 674, 675, 676, 677, 678, 679, 680, 681, 682, 683, 684, 685, 686, 687, 688, 689, 690, 691, 692, 693, 694, 695, 696, 697, 698, 699, 700, 701, 702, 703, 704, 705, 706, 707, 708, 709, 710, 711, 712, 713, 714, 715, 716, 717, 718, 719, 720, 721, 722, 723, 724, 725, 726, 727, 728, 729, 730, 731, 732, 733, 734, 735, 736, 737, 738, 739, 740, 741, 742, 743, 744, 745, 746, 747, 748, 749, 750, 751, 752, 753, 754, 755, 756, 757, 758, 759, 760, 761, 762, 763, 764, 765, 766, 767, 768, 769, 770, 771, 772, 773, 774, 775, 776, 777, 778, 779, 780, 781, 782, 783, 784, 785, 786, 787, 788, 789, 790, 791, 792, 793, 794, 795, 796, 797, 798, 799, 800, 801, 802, 803, 804, 805, 806, 807, 808, 809, 810, 811, 812, 813, 814, 815, 816, 817, 818, 819, 820, 821, 822, 823, 824, 825, 826, 827, 828, 829, 830, 831, 832, 833, 834, 835, 836, 837, 838, 839, 840, 841, 842, 843, 844, 845, 846, 847, 848, 849, 850, 851, 852, 853, 854, 855, 856, 857, 858, 859, 860, 861, 862, 863, 864, 865, 866, 867, 868, 869, 870, 871, 872, 873, 874, 875, 876, 877, 878, 879, 880, 881, 882, 883, 884, 885, 886, 887, 888, 889, 890, 891, 892, 893, 894, 895, 896, 897, 898, 899, 900, 901, 902, 903, 904, 905, 906, 907, 908, 909, 910, 911, 912, 913, 914, 915, 916, 917, 918, 919, 920, 921, 922, 923, 924, 925, 926, 927, 928, 929, 930, 931, 932, 933, 934, 935, 936, 937, 938, 939, 940, 941, 942, 943, 944, 945, 946, 947, 948, 949, 950, 951, 952, 953, 954, 955, 956, 957, 958, 959, 960, 961, 962, 963, 964, 965, 966, 967, 968, 969, 970, 971, 972, 973, 974, 975, 976, 977, 978, 979, 980, 981, 982, 983, 984, 985, 986, 987, 988, 989, 990, 991, 992, 993, 994, 995, 996, 997, 998, 999, 1000, 1001, 1002, 1003, 1004, 1005, 1006, 1007, 1008, 1009, 1010, 1011, 1012, 1013, 1014, 1015, 1016, 1017, 1018, 1019, 1020, 1021, 1022, 1023, 1024, 1025, 1026, 1027, 1028, 1029, 1030, 1031, 1032, 1033, 1034, 1035, 1036, 1037, 1038, 1039, 1040, 1041, 1042, 1043, 1044, 1045, 1046, 1047, 1048, 1049, 1050, 1051, 1052, 1053, 1054, 1055, 1056, 1057, 1058, 1059, 1060, 1061, 1062, 1063, 1064, 1065, 1066, 1067, 1068, 1069, 1070, 1071, 1072, 1073, 1074, 1075, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083, 1084, 1085, 1086, 1087, 1088, 1089, 1090, 1091, 1092, 1093, 1094, 1095, 1096, 1097, 1098, 1099, 1100, 1101, 1102, 1103, 1104, 1105, 1106, 1107, 1108, 1109, 1110, 1111, 1112, 1113, 1114, 1115, 1116, 1117, 1118, 1119, 1120, 1121, 1122, 1123, 1124, 1125, 1126, 1127, 1128, 1129, 1130, 1131, 1132, 1133, 1134, 1135, 1136, 1137, 1138, 1139, 1140, 1141, 1142, 1143, 1144, 1145, 1146, 1147, 1148, 1149, 1150, 1151, 1152, 1153, 1154, 1155, 1156, 1157, 1158, 1159, 1160, 1161, 1162, 1163, 1164, 1165, 1166, 1167, 1168, 1169, 1170, 1171, 1172, 1173, 1174, 1175, 1176, 1177, 1178, 1179, 1180, 1181, 1182, 1183, 1184, 1185, 1186, 1187, 1188, 1189, 1190, 1191, 1192, 1193, 1194, 1195, 1196, 1197, 1198, 1199, 1200, 1201, 1202, 1203, 1204, 1205, 1206, 1207, 1208, 1209, 1210, 1211, 1212, 1213, 1214, 1215, 1216, 1217, 1218, 1219, 1220, 1221, 1222, 1223, 1224, 1225, 1226, 1227, 1228, 1229, 1230, 1231, 1232, 1233, 1234, 1235, 1236, 1237, 1238, 1239, 1240, 1241, 1242, 1243, 1244, 1245, 1246, 1247, 1248, 1249, 1250, 1251, 1252, 1253, 1254, 1255, 1256, 1257, 1258, 1259, 1260, 1261, 1262, 1263, 1264, 1265, 1266, 1267, 1268, 1269, 1270, 1271, 1272, 1273, 1274, 1275, 1276, 1277, 1278, 1279, 1280, 1281, 1282, 1283, 1284, 1285, 1286, 1287, 1288, 1289, 1290, 1291, 1292, 1293, 1294, 1295, 1296, 1297, 1298, 1299, 1300, 1301, 1302, 1303, 1304, 1305, 1306, 1307, 1308, 1309, 1310, 1311, 1312, 1313, 1314, 1315, 1316, 1317, 1318, 1319, 1320, 1321, 1322, 1323, 1324, 1325, 1326, 1327, 1328, 1329, 1330, 1331, 1332, 1333, 1334, 1335, 1336, 1337, 1338, 1339, 1340, 1341, 1342, 1343, 1344, 1345, 1346, 1347, 1348, 1349, 1350, 1351, 1352, 1353, 1354, 1355, 1356, 1357, 1358, 1359, 1360, 1361, 1362, 1363, 1364, 1365, 1366, 1367, 1368, 1369, 1370, 1371, 1372, 1373, 1374, 1375, 1376, 1377, 1378, 1379, 1380, 1381, 1382, 1383, 1384, 1385, 1386, 1387, 1388, 1389, 1390, 1391, 1392, 1393, 1394, 1395, 1396, 1397, 1398, 1399, 1400, 1401, 1402, 1403, 1404, 1405, 1406, 1407, 1408, 1409, 1410, 1411, 1412, 1413, 1414, 1415, 1416, 1417, 1418, 1419, 1420, 1421, 1422, 1423, 1424, 1425, 1426, 1427, 1428, 1429, 1430, 1431, 1432, 1433, 1434, 1435, 1436, 1437, 1438, 1439, 1440, 1441, 1442, 1443, 1444, 1445, 1446, 1447, 1448, 1449, 1450, 1451, 1452, 1453, 1454, 1455, 1456, 1457, 1458, 1459, 1460],
"QUALITY":[17, 3, 5, 31, 14, 32, 21, 35, 25, 19, 23, 32, 45, 8, 7, 7, 6, 17, 34, 25, 39, 66, 65, 51, 62, 51, 22, 9, 5, 7, 1, 5, 1, 4, 11, 71, 64, 34, 30, 34, 30, 24, 26, 29, 30, 27, 47, 46, 24, 30, 18, 13, 8, 6, 10, 1, 8, 14, 9, 28, 24, 20, 8, 20, 24, 6, 8, 15, 5, 31, 7, 2, 10, 9, 25, 1, 34, 9, 16, 25, 12, 34, 22, 28, 26, 28, 31, 50, 21, 24, 30, 48, 36, 62, 46, 40, 7, 12, 2, 16, 1, 3, 15, 36, 18, 34, 11, 28, 19, 20, 9, 48, 25, 24, 10, 15, 19, 28, 17, 19, 48, 49, 9, 19, 29, 30, 11, 19, 10, 6, 1, 5, 13, 15, 21, 25, 22, 13, 14, 12, 16, 8, 7, 11, 6, 10, 8, 14, 11, 21, 17, 5, 9, 1, 26, 15, 14, 1, 1, 3, 3, 1, 1, 6, 2, 4, 3, 10, 8, 12, 21, 17, 15, 22, 56, 41, 36, 33, 27, 33, 30, 32, 54, 1, 7, 21, 19, 3, 20, 23, 21, 32, 15, 6, 1, 1, 9, 12, 12, 29, 26, 19, 44, 36, 46, 19, 11, 17, 12, 17, 15, 4, 22, 24, 39, 32, 22, 10, 3, 5, 9, 10, 49, 57, 33, 56, 52, 45, 40, 14, 9, 35, 6, 18, 16, 7, 12, 3, 2, 10, 21, 47, 35, 38, 53, 12, 9, 27, 20, 31, 26, 19, 3, 11, 12, 59, 8, 1, 2, 19, 5, 5, 23, 40, 10, 5, 4, 9, 14, 54, 40, 15, 18, 18, 31, 31, 21, 31, 37, 24, 24, 19, 20, 7, 2, 9, 30, 40, 21, 8, 22, 31, 16, 31, 31, 16, 17, 24, 12, 38, 2, 9, 10, 25, 24, 21, 41, 43, 52, 13, 13, 8, 12, 9, 14, 15, 16, 15, 40, 41, 30, 19, 23, 16, 19, 26, 28, 34, 31, 38, 8, 26, 18, 16, 20, 15, 18, 6, 3, 35, 22, 3, 7, 5, 6, 9, 13, 15, 26, 36, 51, 64, 67, 65, 11, 58, 3, 1, 14, 57, 22, 53, 34, 38, 16, 2, 15, 21, 21, 9, 31, 27, 16, 19, 17, 24, 31, 28, 30, 55, 38, 12, 12, 9, 19, 4, 12, 14, 15, 19, 1, 1, 22, 8, 7, 16, 4, 7, 11, 3, 6, 10, 4, 3, 12, 23, 14, 39, 50, 31, 15, 28, 33, 31, 39, 28, 18, 4, 11, 5, 2, 21, 54, 64, 45, 66, 27, 30, 13, 7, 1, 37, 54, 15, 15, 20, 29, 18, 13, 32, 24, 21, 14, 13, 12, 1, 8, 8, 13, 40, 38, 20, 22, 22, 22, 45, 20, 42, 23, 26, 5, 2, 9, 7, 12, 10, 13, 18, 6, 15, 8, 5, 4, 16, 3, 13, 7, 7, 1, 12, 1, 1, 14, 20, 15, 17, 21, 17, 42, 27, 9, 1, 4, 5, 6, 6, 19, 1, 14, 1, 1, 9, 1, 7, 14, 28, 40, 13, 31, 10, 2, 15, 62, 33, 24, 11, 23, 31, 24, 18, 13, 35, 10, 9, 1, 9, 20, 22, 14, 43, 21, 7, 9, 30, 11, 24, 19, 17, 29, 28, 10, 14, 9, 14, 15, 15, 1, 21, 26, 23, 57, 48, 11, 18, 25, 1, 13, 53, 22, 1, 15, 11, 4, 5, 13, 3, 18, 66, 49, 25, 14, 24, 16, 18, 7, 22, 42, 30, 37, 44, 35, 22, 11, 14, 1, 20, 12, 12, 11, 2, 10, 9, 6, 34, 16, 20, 17, 5, 7, 29, 35, 20, 14, 9, 5, 12, 32, 17, 10, 1, 4, 1, 20, 9, 28, 41, 33, 40, 34, 13, 7, 7, 11, 16, 18, 31, 41, 59, 61, 33, 22, 7, 2, 10, 15, 9, 14, 4, 10, 9, 15, 15, 16, 20, 29, 34, 40, 30, 20, 29, 20, 46, 14, 27, 22, 30, 31, 1, 10, 10, 12, 8, 19, 12, 25, 21, 11, 1, 21, 3, 1, 1, 7, 6, 8, 10, 11, 17, 31, 13, 11, 7, 7, 25, 27, 30, 58, 10, 15, 15, 10, 11, 10, 8, 12, 35, 26, 6, 1, 52, 24, 21, 17, 14, 46, 51, 36, 17, 26, 32, 39, 32, 23, 9, 5, 19, 8, 9, 2, 8, 1, 6, 28, 1, 1, 1, 4, 12, 26, 37, 32, 35, 36, 29, 37, 17, 7, 16, 9, 34, 29, 25, 12, 19, 14, 14, 31, 16, 24, 48, 54, 55, 61, 33, 27, 25, 19, 1, 6, 17, 37, 9, 11, 1, 12, 18, 1, 4, 8, 8, 5, 7, 13, 33, 74, 50, 44, 17, 27, 25, 24, 15, 39, 47, 22, 23, 11, 21, 47, 11, 1, 8, 1, 11, 15, 15, 33, 20, 20, 18, 17, 27, 30, 29, 26, 35, 24, 31, 39, 22, 12, 10, 15, 5, 18, 8, 11, 12, 23, 4, 5, 3, 7, 6, 1, 3, 4, 4, 9, 45, 16, 19, 14, 10, 17, 8, 16, 5, 10, 16, 6, 1, 2, 20, 26, 39, 50, 30, 18, 13, 4, 7, 3, 21, 28, 26, 12, 2, 13, 14, 6, 9, 6, 15, 17, 12, 9, 5, 6, 9, 1, 2, 21, 2, 1, 6, 12, 34, 15, 21, 25, 53, 60, 58, 48, 52, 33, 42, 54, 61, 54, 48, 57, 10, 4, 17, 9, 9, 6, 1, 2, 9, 37, 28, 29, 59, 43, 39, 62, 48, 52, 35, 16, 13, 10, 13, 16, 12, 8, 21, 22, 1, 3, 15, 10, 21, 23, 40, 34, 36, 45, 42, 23, 28, 23, 19, 8, 16, 20, 11, 27, 19, 22, 16, 9, 31, 42, 28, 25, 19, 1, 6, 24, 12, 24, 25, 68, 56, 36, 49, 12, 33, 38, 30, 11, 23, 16, 16, 1, 14, 20, 27, 17, 37, 27, 21, 2, 2, 7, 18, 8, 33, 17, 9, 3, 1, 17, 27, 29, 44, 42, 64, 53, 66, 68, 41, 36, 22, 38, 14, 20, 4, 10, 14, 14, 4, 34, 36, 17, 15, 17, 13, 14, 14, 24, 34, 57, 43, 41, 22, 4, 12, 10, 19, 10, 35, 37, 20, 15, 9, 19, 23, 19, 41, 7, 32, 40, 67, 74, 54, 25, 43, 29, 32, 11, 9, 4, 13, 10, 15, 17, 14, 19, 36, 55, 24, 20, 30, 46, 24, 36, 3, 17, 23, 38, 45, 1, 14, 9, 39, 16, 18, 57, 72, 66, 22, 17, 43, 30, 38, 36, 21, 26, 13, 12, 26, 11, 2, 15, 14, 1, 1, 10, 15, 12, 20, 16, 34, 38, 50, 17, 6, 4, 1, 1, 12, 24, 9, 13, 19, 12, 25, 13, 26, 4, 8, 8, 24, 15, 29, 16, 13, 40, 24, 26, 50, 24, 14, 21, 2, 9, 6, 7, 9, 1, 3, 22, 7, 11, 20, 29, 25, 45, 38, 18, 23, 27, 21, 23, 19, 7, 18, 14, 16, 25, 21, 5, 5, 21, 41, 13, 29, 11, 4, 17, 26, 46, 19, 18, 18, 19, 10, 17, 22, 31, 15, 7, 1, 4, 20, 14, 49, 32, 20, 2, 7, 22, 9, 32, 32, 44, 22, 21, 26, 16, 9, 24, 14, 13, 9, 23, 16, 10, 17, 45, 30, 11, 5, 20, 17, 25, 23, 35, 12, 31, 26, 36, 20, 38, 17, 17, 27, 9, 24, 15, 18, 9, 14, 24, 12, 15, 7, 6, 6, 16, 16, 17, 9, 18, 14, 20, 13, 14, 28, 10, 10, 6, 9, 22, 19, 17, 21, 10, 14, 15, 7, 9, 10, 15, 32, 5, 1, 5, 16, 9, 8, 24, 19, 21, 16, 16, 38, 45, 49, 30, 43, 25, 12, 31, 38, 28, 29, 22, 44, 28, 42, 27, 67, 41, 21, 19, 45, 8, 39, 17, 7, 16, 13, 8, 33, 32, 14, 5, 38, 24, 21, 28, 40, 49, 74, 74, 32, 23, 9, 43, 24, 64, 56, 30, 38, 45, 33, 1, 1, 1, 1, 18, 23, 10, 26, 29, 12, 14, 28, 15, 16, 20, 33, 18, 5, 1, 18, 23, 39, 35, 30, 44, 41, 28, 33, 73, 72, 64, 29, 35, 28, 6, 15, 11, 21, 20, 34, 36, 11, 19, 6, 1, 6, 14, 15, 51, 46, 49, 46, 41, 51, 30, 24, 31, 17, 6, 9, 15, 16, 9, 11, 14, 14, 25, 39, 43, 30, 25, 43, 53, 12, 22, 50, 19, 5, 4, 10, 8, 10, 11, 29, 66, 46, 32, 70, 69, 68, 31, 40, 30, 20, 14, 17, 9, 26, 26, 32, 23, 8, 12, 24, 49, 30, 21, 21, 52, 54, 41, 29, 12, 6, 8, 1, 8, 14, 5, 32, 21, 16, 14, 20, 18, 12, 14, 26, 15, 1, 1, 1, 1, 1, 1, 2, 1, 1],
"RD_FILTER":"SRA_READ_FILTER_PASS",
"READ":"AGATTTAATCCAAGAGAAGCAAAAAAAGCTCTATAAAATATCATGCAATTACATTACCATCGCTTCAACTGCGCTCTAATAACGTCAGCTTTGGCTAATTACTTCCTTTTCCCAATATACCCTTTTCCTGAGGTACCATTGGTTTCATGCCAATCCTGTCAGAGTGAACAGATAAATATAGCCAATTGTGATACGTTAGTCTGGAACTTAACTATTGGTAGTTTGAAGACTGGTGCTACGTTAATGTTAAAAGGAATCAACGGGTCACTAACTATTTAACGCTACAAATTTAACCTCATTCGCTGCTGATCCTGGAGTCCAACACAGATTCTTTTGAACCCTACAGAGTTTGACAATCTTGACTTCTTGCTTCATTTTGGGTTTTACAGAGCTGAATTATAAGCTATAAACTGATTACTTCCCCGCCATCTCCCTCCAGTTTACTTCACAATATAAATACACTGCTAACATTTATTCCGACACCACTTTATACAATCTGTCGATGGATTCAGCCTAAAAGTTAACGTGTTTCAACGTCAATATATACTGAAATTAACCCATTGATCAAATCTCCAGTTGAACAGCTCAGCGATTTTTACAATTCGTTTTACAACGGTAACCAGACTAAAATCACCTTGGTGACTTGGTTTGGGCAAACATATCAGTTTGACCGATGTCCACTCTGTTTCCTTCGTTAACTTGCAAAAGATTAACTTCATTAGGTCACTTTTAACATTTCCATCTCAAGTTTGAATTTCTAAGCTAACACCGCCAAACCTTCAGTATCGTTTCAATGACTACTTGAAGAACTTTGTCGTTCTCTTCGAATTGTCAACCACCATAGGTGGTGTGCTTGTCGTCGCTAATATTAGGTCTTTACAAAAAATTGGTGGTTCGACCAACCTAACAACCATTGGCAGTACTTTGGTAGTTGTTATTGGTAACTTTTTCTCCTGAACCTAGACTTTCTTTGAAGTTGGTGGCGCAGATGTCGAATAAGTCAAGCAATTTCCTGTAATGCTTTGAAAGCTTTTGCAAAAAGAAAGCTATCAAGGGTGAATCTTTTGTTTGCAAAATGGTGCTACCCACATACTTGTTAAACTCGCCTTTCCAATACAATCAAGCCAAACTACTGCCAAGGTTTTCCAAGTCATTTAAGGCCGGGAAAGAAGGTTTTGCGATATCAAGGCTGCTGCTTTCTGCCCCTCTAGTGTTTTAGTTTCTAGCAATTCCAGCTCTAGCTTAAGAGAGTTCCGACAATGCCGCTATCATGGCACCAATTGGCCAAACAACCCTTTGGTCGGTTTTTTGACGGCAATCATCATGTTCTATAATGTAATAGAATGAAGAAAAATTCTCATTTTGACTAGTACCTGTCATCACGACACGAACAAATAAACATTTATTTAAAAATTTTATGTATTCAAATATTTTTCGTGGGAAAGGATAAAGTAAACCAA",
"READ_FILTER":"SRA_READ_FILTER_PASS",
"READ_LEN":1460,
"READ_NUMBER":0,
"READ_SEG":[[0, 1460]],
"READ_START":0,
"READ_TYPE":"SRA_READ_TYPE_BIOLOGICAL",
"SIGNAL_LEN":0,
"SPOT_COUNT":1,
"SPOT_FILTER":1,
"SPOT_GROUP":"",
"SPOT_ID":1,
"SPOT_IDS_FOUND":"",
"SPOT_LEN":1460,
"TRIM_LEN":1460,
"TRIM_START":0,
"X":"",
"Y":""
}
]
